docs/configuration/index.md

```diff
@@ -1370,7 +1370,7 @@ These Historical configurations can be defined in the `historical/runtime.proper
 
 |Property|Description|Default|
 |--------|-----------|-------|
-|`druid.server.maxSize`|The maximum number of bytes-worth of segments that the process wants assigned to it. This is not a limit that Historical processes actually enforces, just a value published to the Coordinator process so it can plan accordingly.|0|
+|`druid.server.maxSize`|The maximum number of bytes-worth of segments that the process wants assigned to it. The Coordinator process will attempt to assign segments to a Historical process only if this property is greater than the total size of segments served by it. Since this property defines the upper limit on the total segment size that can be assigned to a Historical, it needs to be set to a value greater than zero.|0|
```

Comment:
Should this mention that this size should be equal(ish) to the sum of all segment cache location sizes? Ideally someday we can drop this property and just compute it from the segment cache config, but .. that day is not today, so maybe it is worth including.